### PR TITLE
[Cosmos] Update code owners list

### PR DESCRIPTION
**Pull request by mccoyp**
# Description

This resolves a code owner [linting failure](https://dev.azure.com/azure-sdk/public/_build/results?buildId=4119658&view=logs&j=a460d732-23aa-5493-a411-cc406067acf8&t=8c98ab76-57b2-59d0-3d3a-18dce788f3ba).

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by xiangyan99 — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -230,7 +230,7 @@
 /sdk/cosmos/                                                         @annatisch @Azure/azure-cosmos-python-sdk
 
 # ServiceLabel: %Cosmos %Service Attention
-#/<NotInRepo>/                                                       @pjohari-ms @simorenoh @AbhinavTrips @bambriz @pilchie
+#/<NotInRepo>/                                                       @pjohari-ms @simorenoh @AbhinavTrips @pilchie
```
Comment (xiangyan99) [inline .github/CODEOWNERS:233]: You can use ServiceOwners instead: e.g. https://github.com/Azure/azure-sdk-for-python/edit/main/.github/CODEOWNERS#L47

**Comment by kristapratico** [comment]
Should this person be deleted, or do they just need to make their Azure org public?

**Comment by JimSuplizio** [comment]
> Should this person be deleted, or do they just need to make their Azure org public?

[Their Microsoft and Azure org membership should be public](https://eng.ms/docs/products/azure-developer-experience/onboard/access).

**Comment by mccoyp** [comment]
I opened this PR pre-emptively in case Bryan had left the team, but he just got back to me and said the loss of access was due to a GitHub account issue. He should be getting write access back soon, so I'll close this PR for now.

**Pull request closed without merging**